addons/mail/static/src/core/common/thread.js

```diff
@@ -368,14 +368,11 @@ export class Thread extends Component {
         this.threadService.fetchMoreMessages(this.props.thread);
     }
 
-    async jumpToPresent(behavior) {
+    async jumpToPresent() {
         this.messageHighlight?.clearHighlight();
         await this.threadService.loadAround(this.props.thread);
         this.props.thread.loadNewer = false;
-        this.present.el?.scrollIntoView({
-            behavior: behavior ?? (this.props.order === "asc" ? "smooth" : "instant"), // FIXME somehow smooth not working in desc mode
-            block: "center",
-        });
+        this.props.thread.scrollTop = "bottom";
```

Comment:
Shouldn't it be either `bottom` or `0` depending on the thread order?

Reply:
And if so, it would be nice to have a test for that case too (jump to present in chatter, I guess)